userguide/tasks.rst

```diff
@@ -282,48 +284,47 @@ task.
-   |                                  |                             | system. The user name cannot contain spaces or exceed 17                                  |
-   |                                  |                             | characters.                                                                               |
+   | User                             | drop-down menu              | Select the user to run the rsync task. The user selected must have permissions to write   |
+   |                                  |                             | to the specified directory on the remote host.                                            |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
    | Remote Host                      | string                      | Enter the IP address or hostname of the remote system that will store the copy. Use the   |
    |                                  |                             | format *username@remote_host* if the username differs on the remote host.                 |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-   | Remote SSH Port                  | integer                     | Only available in  *Rsync over SSH* mode. Allows specifying an SSH port                   |
+   | Remote SSH Port                  | integer                     | Only available in  *Rsync over SSH mode*. Allows specifying an SSH port                   |
    |                                  |                             | other than the default of *22*.                                                           |
    |                                  |                             |                                                                                           |
    +----------------------------------+-----------------------------+-------------------------------------------------------------------------------------------+
-   | Rsync mode                       | drop-down menu              | Choices are *Rsync module* or                                                             |
-   |                                  |                             | *Rsync over SSH*.                                                                         |
+   | Rsync mode                       | drop-down menu              | The choices are *Rsync Module mode* or *Rsync over SSH mode*                              |
```

Comment:
As above, "mode" should not be included in the italics.

Reply:
done